lru_cache/lru_cache.py

```diff
@@ -33,4 +43,19 @@ def get(self, key):
     the newly-specified value.
     """
     def set(self, key, value):
-        pass
+        # If the key allredy in hashing_table then
+        if key in self.hashing_table:
+            # set the key value to input value
+            self.hashing_table[key] = value
+            return # stop est here
+        # if there are more items held than the limit
+        if self.held >= self.limit:
+            # then del oldest cached value
+            del self.hashing_table[self.linked_list.tail.value]
```

Comment:
Nice of you to use del keyword and actively delete key-value pair from the dictionary.